src/nbl/video/vulkan/profiles/NablaCore.json

```diff
@@ -1,2017 +1,2216 @@
+                        "maxFragmentDualSrcAttachments": 1,
+                        "maxFragmentInputComponents": 116,
+                        "maxFragmentOutputAttachments": 8,
+                        "maxFramebufferHeight": 16384,
+                        "maxFramebufferLayers": 256,
+                        "maxFramebufferWidth": 16384,
+                        "maxGeometryInputComponents": 0,
+                        "maxGeometryOutputComponents": 0,
+                        "maxGeometryOutputVertices": 0,
+                        "maxGeometryShaderInvocations": 0,
+                        "maxGeometryTotalOutputComponents": 0,
+                        "maxImageArrayLayers": 2048,
+                        "maxImageDimension1D": 16384,
+                        "maxImageDimension2D": 16384,
+                        "maxImageDimension3D": 2048,
+                        "maxImageDimensionCube": 16384,
+                        "maxInterpolationOffset": 0.4375,
+                        "maxMemoryAllocationCount": 16384,
+                        "maxPerStageDescriptorInputAttachments": 7,
+                        "maxPerStageDescriptorSampledImages": 96,
+                        "maxPerStageDescriptorSamplers": 16,
+                        "maxPerStageDescriptorStorageBuffers": 31,
+                        "maxPerStageDescriptorStorageImages": 8,
+                        "maxPerStageDescriptorUniformBuffers": 15,
+                        "maxPerStageResources": 127,
+                        "maxPushConstantsSize": 128,
+                        "maxSampleMaskWords": 1,
+                        "maxSamplerAllocationCount": 1024,
+                        "maxSamplerAnisotropy": 16,
+                        "maxSamplerLodBias": 4.0,
+                        "maxStorageBufferRange": 1073741820,
+                        "maxTessellationControlPerPatchOutputComponents": 120,
+                        "maxTessellationControlPerVertexInputComponents": 124,
+                        "maxTessellationControlPerVertexOutputComponents": 124,
+                        "maxTessellationControlTotalOutputComponents": 2048,
+                        "maxTessellationEvaluationInputComponents": 124,
+                        "maxTessellationEvaluationOutputComponents": 124,
+                        "maxTessellationGenerationLevel": 64,
+                        "maxTessellationPatchSize": 32,
+                        "maxTexelBufferElements": 67108864,
+                        "maxTexelGatherOffset": 7,
+                        "maxTexelOffset": 7,
+                        "maxUniformBufferRange": 65536,
+                        "maxVertexInputAttributeOffset": 2047,
+                        "maxVertexInputAttributes": 29,
+                        "maxVertexInputBindingStride": 2048,
+                        "maxVertexInputBindings": 31,
+                        "maxVertexOutputComponents": 124,
+                        "maxViewports": 1,

```

Comment:
see if we ever assumed more than 1 or that multiview will always be available

Reply:
we require multiview but there's no minimum number hardcoded